nusacrowd/nusa_datasets/wikilingua/wikilingua.py

```diff
@@ -0,0 +1,139 @@
+import os
+import pickle
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from nusantara.utils import schemas
```

Comment:
I think since the folder `nusantara` had been changed into `nusacrowd` can change this line too to
```suggestion
from nusacrowd.utils import schemas
```